docs/vlab/overview.md

```diff
@@ -71,6 +74,30 @@ INFO: /dev/kvm exists
 KVM acceleration can be used
 ```
 
+### Oras
+
+For convenience Hedgehog provides a script to install `oras`:
+
+```bash
+curl -fsSL https://i.hhdev.io/oras | bash
+```
+
+### Hhfab
+
+Hedgehog maintains a utility to install and configure VLAB, called `hhfab`.
+
+You need a GitHub access token to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once in possession of the credentials, use the provided username and token to log into the GitHub container registry:
+
+```bash
+docker login ghcr.io --username provided_username --password provided_token
+```
+
+Once logged in, download and run the script:
+
+```bash
+curl -fsSL https://i.hhdev.io/hhfab | bash
```

Comment:
Side note, we may have to change this domain in the future (https://every.to/p/the-disappearance-of-an-internet-domain).

Reply:
looks like we're good for a few years with it, but that's a danger of ccTLDs :(